v20.1/build-a-spring-app-with-cockroachdb-jpa.md

```diff
@@ -622,7 +622,7 @@ Typically, Liquibase properties are defined in a separate [`liquibase.properties
 
 `Account.java` defines the [entity](https://en.wikipedia.org/wiki/Domain-driven_design#Building_blocks) for the `accounts` table. This class is used throughout the application to represent a row of data in the `accounts` table.
 
-Here are the contents of [`Account.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/Account.java):
+Here are the contents of [`Account.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/jpa/Account.java):
```

Comment:
The url is now actually 

https://github.com/cockroachlabs/roach-data/tree/master/roach-data-jpa/src/main/java/io/roach/data/json/Account.java